.travis.yml

```diff
@@ -8,22 +8,18 @@ sudo: false
 
 cache:
   directories:
-    - ${HOME}/.m2/repository
+    - ${HOME}/.m2
 
 before_install:
   - if [ ! -z "${GPG_SECRET_KEYS}" ]; then echo ${GPG_SECRET_KEYS} | base64 --decode | ${GPG_EXECUTABLE} --import; fi
   - if [ ! -z "${GPG_OWNERTRUST}" ]; then echo ${GPG_OWNERTRUST} | base64 --decode | ${GPG_EXECUTABLE} --import-ownertrust; fi
 
 install:
-  - mvn --settings .travis/settings.xml --show-version dependency:resolve
-  - mvn --settings .travis/settings.xml --show-version dependency:resolve-plugins
+  - mvn -T 2.0C --settings .travis/settings.xml --show-version dependency:resolve
+  - mvn -T 2.0C --settings .travis/settings.xml --show-version dependency:resolve-plugins
```

Comment:
Based on information from https://cwiki.apache.org/confluence/display/MAVEN/Parallel+builds+in+Maven+3

Reply:
Note that the Travis machines we run on have 2 cores:

<img width="688" alt="screen shot 2018-12-28 at 1 57 59 pm" src="https://user-images.githubusercontent.com/4274030/50525331-c5279580-0aa8-11e9-8d4c-3b2fc701e174.png">